docs/ansible/vars.md

```diff
@@ -258,6 +258,13 @@ node_labels:
   label2_name: label2_value
 ```
 
+  Example of `node_labels` definition in `inventory.ini` format:
+
+```toml
```

Comment:
Can't be toml and ini ^.

Also, IMO that kind of vars is much more readable in yaml.